app/validators/email_format_validator.rb

```diff
@@ -0,0 +1,79 @@
+# frozen_string_literal: true
```

Comment:
Would it be worth adding a comment to say where these rules come from?

Reply:
Good idea - will add 👌